runUItests.sh

```diff
@@ -0,0 +1,26 @@
+# !/bin/sh
+
+# This documents (and performs) necessary commands for local UI testing.
+# Experts might run these manually on demand. For example, repeatedly
+# running the final npx extest command for debugging purposes is normal.
+
+set -e;
+set -x;
+
+cd rascal-vscode-extension
+
+UITESTS=/tmp/vscode-uitests
+
+# cleanup to avoid contamination with previous runs
+
+rm -rf out || true
+rm -rf node_modules || true
+rm -rf $UITESTS || true
```

Comment:
quite nuclear, will cause you to redownload vs code & chromedriver everytime. a bit of a heavy thing.


I think we should have these behind an optional flag or something (like `-c` to clean everything) or otherwise a separate script.